WordPress/twentytwentyfive
# Add pattern: Overlap Images With Paragraph

**Description**
This PR replaces https://github.com/WordPress/twentytwentyfive/pull/252 since the contributor is unavailable.
Because the original PR is on a branch on a fork, moving it to this repository will make it easier to edit for other contributors.

Fixes: https://github.com/WordPress/twentytwentyfive/issues/64
Figma: https://www.figma.com/design/dzGCSntVch4EQdVERTqyVK/Twenty-Twenty-Five?node-id=3177-1817&t=UAEPVUr1BaR1QF5S-0



**Screenshots**
Mobile:

![66 local__p=1856](https://github.com/user-attachments/assets/930c6fb0-d746-49fe-912d-21cf869e7fb9)




**Testing Instructions**
Add the pattern.
Preview the pattern on different browser widths.
Confirm that the pattern matches the design.

------
If you're merging code through a pull request on GitHub, copy and paste the following into the bottom of the merge commit message.

Co-authored-by: shail-mehta <shailu25@git.wordpress.org>
Co-authored-by: juanfra <juanfra@git.wordpress.org>
Co-authored-by: carolinan <poena@git.wordpress.org>
Co-authored-by: beafialho <beafialho@git.wordpress.org>
Co-authored-by: hanneslsm <hanneslsm@git.wordpress.org>
